docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -4,6 +4,8 @@
 Cause Model Template
```

Comment:
My preference for this title would be to actually write "Full Name of Cause in GBD 2017" here instead of "Cause Model Template", in order to make it clear that this .rst is indeed a template, rather than a document about what template to use.

Reply:
The main problem with that approach is that when you build the documentation in Sphinx, the page will show up in the table of contents as "Full Name of Cause in GBD 2017," which would be confusing.

I guess one solution would be to have both a title and a subtitle for the template, e.g.
```
-----------------------
Cause Model Template
-----------------------

(Description of what's in this document --
delete this and everything above once you start writing)

============================
Full Name of Cause in GBD 2017
============================
```

I think that will work because it looks like Sphinx doesn't require the heading symbols to be consistent between documents. Therefore, "Cause Model Template" will render as the title until it is deleted, at which point the "Full Name of Cause in GBD 2017" subtitle will become the document's title (I guess I should test this to make sure).

What do you think of that idea, @beatrixkh ?

Reply:
Ahh I see. In that case--then yes, I like this @NathanielBlairStahn !